webplugin/template/chat.html

```diff
@@ -38,9 +48,21 @@
                                 iframe.classList.add("change-kommunicate-iframe-height");
                             });
                         });
+                        //to show the pre chat lead collection Form if enabled 
+                        var preChatLauncher = window.KommunicateGlobal.window.document.getElementById("km-anonymous-chat-launcher")
+                        if (window.KommunicateGlobal && preChatLauncher) {
+                            var intervalForPreChatLead = setTimeout(function () {
+                                preChatLauncher.click()
+                                window.KommunicateGlobal.window.document.getElementById("km-modal-close").style.display = "none"
```

Comment:
can you add the clearInterval here itself?

Reply:
done